packages/odata-common/src/selectable/field.ts

```diff
@@ -1,25 +1,7 @@
-import BigNumber from 'bignumber.js';
 import { EntityBase, Constructable } from '../entity-base';
-import { Time } from '../time';
-
-/**
- * Union type to represent all possible types of a field.
- * @internal
- */
-export type FieldType =
```

Comment:
Could you please explain more about this change and the corresponding other adjustment?

Reply:
As we not have possibly custom (de-)serializers this types does not make sense anymore. It was already removed in most places, but forgotten here. I should have done this on a separate PR ;)